partcad/src/partcad/part_factory_python.py

```
@@ -7,6 +7,7 @@
 # Licensed under Apache License, Version 2.0.
 #
 
+from . import logging as pc_logging
```

Ruff (0.8.2)

`.logging` imported but unused

Remove unused import: `.logging`

(F401)